### ISSUE TITLE
Action labels of the test split

### ISSUE THREAD

**Issue by fylwen** [taeinkwon/h2odataset#11]
Hi, I wonder whether the action labels of the testing split are available. I can find the action label for the training and validation set, however, I fail to find that for sequences of testing sets.

Thanks for your help!

Best regards,
Frances

**Comment by fylwen** [comment]
Dear Taein,

Hi. Thanks for the information provided!

Best regards,
Frances.

**Comment by fylwen** [comment]
Dear Taein,

Hi, I tried with the CodaLab system to evaluate only the action on the test split, where my submission is with an empty _hand_poses.json_, and found the logs on the webpage indicate an error. Still, I downloaded the output from the scoring step, in the _output_file.zip_ there is a _score.txt_ file showing a score. Is this score in the _score.txt_ the accuracy for my action estimation? 

Thanks a lot for your help!

Best regards,
Frances
